common/plugins/eu.esdihumboldt.hale.common.instance/src/eu/esdihumboldt/hale/common/instance/io/util/EnumWindingOrderTypes.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2016 Data Harmonisation Panel
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     Data Harmonisation Panel <http://www.dhpanel.eu>
+ */
+
+package eu.esdihumboldt.hale.common.instance.io.util;
+
+/**
+ * TODO Type description
+ * 
+ * @author Arun
+ */
+public enum EnumWindingOrderTypes {
+	CLOCKWISE, COUNTERCLOCKWISE, DONTCHANGE;
```

Comment:
I remember to add JavaDocs on Function but with class/ interface/ Enum. I will remember it now onward. :)